include/wsl/stubs/basetsd.h

```diff
@@ -0,0 +1,404 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
`basetsd.h` is not designed equal to windows's `basestd.h`, it's a composite of `windows.h` and `basetd.h`, maybe I should rename it?